routes/api/project/firewall/firewall_rule.rb

```diff
@@ -36,6 +29,13 @@ class CloverApi
         response.status = 204
         r.halt
       end
+
+      request.get true do
+        if firewall_rule
```

Comment:
We check the existence of a single record after retrieving it in other endpoints. Should we follow the same procedure here as well?

https://github.com/ubicloud/ubicloud/blob/fc3cf1352ee5e88fddbadbca110209f758bcdfd3/routes/api/project/firewall.rb#L35